### ISSUE TITLE
Remove embedded Cassandra, RexsterServer

### ISSUE THREAD

**Issue by mbroecheler** [thinkaurelius/titan#368]
from the Titan distribution and associated dependencies. This will make the project leaner.

Add scripting to easily stand up Titan+Rexster, backend service and elastic search on a local machine (see #263 )

**Comment by dalaro** [comment]
RexsterTitanServer is already deleted and rexster-\* dependencies in titan-core have been removed.

I also removed the embedded Cassandra adapter and its tests, but as we recently discussed, I'm going to restore it so that we can provide it without support.

**Comment by dalaro** [comment]
A quickstart distribution with Cassandra, ES, Rexster, Titan, and a titan.sh script that accepts the verbs "start", "stop", and "clean" is in the master @ titan-dist/titan-dist-quickstart and slated for 0.4.0.  I think any remaining engineering on the easymode script and distro are better addressed in #263 and #376.  #376 is also in the 0.4 milestone.

I made .rpm and .deb packages for Titan a few weeks ago.  RPM specfile generation and Debian controlfile generation is integrated with Maven via script resources filtered in the titan-dist module.  This includes logic for partitioning Titan's Maven dependency tree among subpackages that correlate with the major modules, like titan-cassandra, titan-hbase, etc. without jar repetition.  However, the init.d scripts in these packages in particular need changes now that RexsterTitanServer is gone.  The present issue is only tangentially related, so I created #398 to track these init.d script and OS package changes.
